src/vcpkg/install.cpp

```diff
@@ -814,7 +819,12 @@ namespace vcpkg::Install
             }
             else
             {
-                auto msg = msg::format(msgCMakeTargetsUsage, msg::package_name = bpgh.spec.name()).extract_data();
+                // clang-format off
+                // to preserve output lines
+                auto msg = msg::format(msgCMakeTargetsUsage, msg::package_name = bpgh.spec.name()).appendnl()
+                    .append_indent().append(msgCMakeTargetsUsageHueristicMessage).appendnl()
+                    .extract_data();
+                // clang-format on
```

Comment:
I don't think the `clang-format` is necessary, but I'm not that opinionated on it.

Reply:
Without that it formats like this, losing the division between lines of output:

<img width="578" alt="image" src="https://user-images.githubusercontent.com/1544943/163236428-7cd2d964-809f-4817-a63e-43e7d7e61056.png">

I don't feel strongly about it either though.

Reply:
I avoided it in a different way.

Reply:
Maybe we need an `appendnl(message)`?

Reply:
@vicroms The cross product of different appends starts to get pretty big then; I think this is fine